api/tests/routes/pro/patch_draft_offer_test.py

```diff
@@ -97,21 +100,23 @@ def test_patch_draft_offer_without_product_with_new_ean_should_succeed(self, cli
 
     def test_patch_draft_offer_without_product(self, client):
         user_offerer = offerers_factories.UserOffererFactory(user__email="user@example.com")
-        venue = offerers_factories.VirtualVenueFactory(managingOfferer=user_offerer.offerer)
+        venue = offerers_factories.VenueFactory(managingOfferer=user_offerer.offerer)
         offer = offers_factories.OfferFactory(
             name="Name",
             subcategoryId=subcategories.LIVRE_PAPIER.id,
             venue=venue,
             description="description",
-            url="http://example.com/offer",
+            offererAddress=offerers_factories.OffererAddressFactory(offerer=user_offerer.offerer),
```

Comment:
nitpik : tu veux pas sortir `offerer_address = offerers_factories.OffererAddressFactory(offerer=user_offerer.offerer)`
pour avoir ici 
```suggestion
            offererAddress=offerer_address,
```
?